pre-meeting checklist for sprint week 2020

humnaawan opened the pull request.

humnaawan committed:
copy of the index for the july meeting.

humnaawan committed:
checklist for the 2020 sprint week.

humnaawan committed:
updated link to the desc kernels.

rmandelb commented on index.html:
```diff
@@ -151,7 +170,8 @@ <h2>Access meeting resources</h2>
           <div class="text">Check out the
-               <a href="https://confluence.slac.stanford.edu/x/w4DyE" title="link to Confluence">participant guidelines</a>
+               <a href="https://confluence.slac.stanford.edu/x/zJV7EQ"
+               title="link to Confluence">participant guidelines</a>
```
Are the "participant guidelines" relevant for sprint week?  They seem to be more for plenary/parallels at a collaboration meeting.  In that case, maybe we don't need them for this checklist?

humnaawan replied:
so the MAC is working on updating them but its likely that they wont be updated before tomorrow. i can remove the item from the checklist and perhaps mention it during the sprint week.

rmandelb commented on index.html:
```diff
@@ -189,17 +219,23 @@ <h2>Prepare for engaging during the meeting</h2>
-                   <a href="https://lsstc.slack.com/messages/C90AMAG80/" title="link to LSSTC Slack">#desc-collab-meeting</a> to hear meeting announcements and to interact with meeting participants.
+                   <a href="https://lsstc.slack.com/messages/C2M8FG5A6"
+                   title="link to LSSTC Slack">#desc-hack-sprint</a>
+                    to hear announcements and to interact with with all the sprinters.
```
```suggestion
                    to hear announcements and to interact with all the sprinters.
```

rmandelb commented on index.html:
```diff
@@ -189,17 +219,23 @@ <h2>Prepare for engaging during the meeting</h2>
                <li>
-                  <a href="https://app.slack.com/client/T06D204F2/C0176US657B/details" title="link to LSSTC Slack">#desc-collab-meeting-hotline</a> to report any technical problems that may arise during the meeting.
+                  <a href="https://app.slack.com/messages/C01FAP8T1PB"
+                  title="link to LSSTC Slack">#desc-sprint-science-help</a>
```
I wonder if this might be more slack channels than we need?  I would imagine, for example, that some questions might more naturally be asked in pre-existing ones, like a CCL question in #desc-ccl and a DC2 question in #desc-dc2-users, and then the few science questions with no obvious home could go in #desc-hack-sprint?  (It might even be easier for people not attending synchronously to engage with questions in those other channels?)

humnaawan replied:
so the thinking within the LOC was that #desc-hack-sprint might be too busy with announcements etc. s.t. new members who want to get science help might not want to post their inquiries these. i can ask the LOC re further thoughts on this.

rmandelb commented on index.html:
```diff
@@ -216,12 +252,24 @@ <h2>Computing resources etc.</h2>
+                    </li>
+               </ul>
+               Also, make sure you are in the "lsst" group. If not, contact
+               Richard Dubois or Debbie Bard to request to be added to the "lsst" group.
```
I suggest indicating how to contact Richard or Debbie (e.g., suggest using slack and give their slack handles).

rmandelb replied:
Also I just checked in with Richard because confluence said something slightly different.  He told me that both Debbie Bard and Heather Kelly are co-PIs on the allocation, so actually all 3 of them can be listed as contacts here.

rmandelb reviewed (approved):
Hi - Thanks for this update!  I looked at the diff and at the rendered checklist, and have some minor suggestions for your consideration (see review comments on the diff).

Also, here are two comments on items that don't show up in the diff for `index.html` since they hasn't changed since the last version:

1. I noticed the form lists Alex Kim as a contact/maintainer for this list and encourages emailing feedback to him.  Since @plazas  is now the meetings committee chair, I wonder if it should be updated to direct feedback to him?

2. Regarding pronouns: After the July collaboration meeting, Slack was updated to enable pronouns to be listed in user profiles, rather than in names. Since this option was added, I wonder if the item about adding pronouns on slack should be made more specific - i.e., do we particularly recommend that option of having pronouns in the profile or recommend having them in the name or ...?  And might we provide a link to a quick resource for somebody who isn't sure why they might want to do this? (This could be as simple as turning the word "pronouns" into a link to https://confluence.slac.stanford.edu/display/LSSTDESC/Personal+Pronouns)

Since my suggestions are minor ones for your consideration, I am clicking the "Approve" option rather than "Request changes", so that my review won't block you from merging when you've addressed any of my comments that you find helpful.

humnaawan committed:
fixed typo.

Co-authored-by: Rachel Mandelbaum <rmandelb@andrew.cmu.edu>

humnaawan committed:
replaced alex's info with andres's.

humnaawan committed:
added details re pronouns.

humnaawan committed:
added heather re NERSC access; added contact info.

humnaawan commented:
thank you @rmandelb! i think i've addressed all the concerns except re participant guidelines and the too-many slack channels; i'll get back re these once i hear from the LOC.

yymao commented on index.html:
```diff
@@ -272,13 +347,13 @@ <h2>Social media</h2>
           <p>If you have questions or suggestions about this page, please
                <a href="https://github.com/LSSTDESC/pre-meeting-checklist/issues" title="link to GitHub">open an issue</a> or contact
-               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:agkim@lbl.gov?Subject=DESC%20Meeting%20Checklist" target="_top">Alex Kim</a>.</p>
+               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:plazasmalagon@gmail.com?Subject=DESC%20Meeting%20Checklist" target="_top">Andrés Plazas</a>.</p>
```
Once published, this website is world-viewable. Please double check with @plazas to make sure it's ok to include email address in a mailto link (which may result in bot collecting this email address).

humnaawan replied:
okay andres is okay with either a slack or an email mention; i've just gone with the former.

yymao reviewed (commented):
This is a technical review on the web elements. I did not review the actual content.  I made one comment to require @plazas's consent in including the email address.

humnaawan committed:
removed the science hotline slack channel.

humnaawan committed:
updated contact info for andres.

humnaawan commented:
okay the LOC would like to keep the participant guidelines even as they are not specific to sprint week; the overarching ideas and considerations would be good to keep in mind.

i think i've addressed all the concerns at this point. i'll merge the PR in the next 30 mins.

humnaawan commented:
@yymao do i need to do something specific to get the weblink to work?
